### PR TITLE
Changed variable declarations to es6 syntax, took advantage of es6 default parameters, and converted str.match to str.test where it made sense to

### PR DESCRIPTION
**Pull request by DovidM**
This is my first 'real' pull request, so please let me know if I'm doing something wrong

### CODE REVIEW

**Comment by kylefarris** [comment]
Thanks @DovidM. This is great (saved me a bit of time). Would you mind changing the callback syntax to arrow functions as well? This would allow us to get rid of some of the `self` and `that` nonsense. If you don't have time for that, I'll accept the PR as is. Just thought it would make this PR more of a "complete" transition to ES6 syntax.

**Comment by DovidM** [comment]
Do you mean like what I just pushed, or something else?

File: README.md [hunk 1]
```diff
@@ -1275,10 +1275,10 @@ const app = express();
 const settings = require('db.json');
 const pool = require('node-querybuilder').QueryBuilder(settings, 'mysql', 'pool');
 
-app.post('/update_account', function(req, res) {
+app.post('/update_account', (req, res) => {
     const user_id = req.session.user_id;
-    const sanitize_name = function(name) { return name.replace(/[^A-Za-z0-9\s'-]+$/,'').trim(); };
-    const sanitize_age = function(age)  { return age.replace(/[^0-9]+$/,'').trim(); };
+    const sanitize_name = name => { return name.replace(/[^A-Za-z0-9\s'-]+$/,'').trim(); };
```
Comment (kylefarris) [inline README.md]: When the arrow function is a single line that returns a value, you can remove the curly braces and return statement.

**Example:**
```javascript
const sanitize_name = name => name.replace(/[^A-Za-z0-9\s'-]+$/,'').trim();
```

Comment (kylefarris) [inline README.md]: Haha, I just realized the comment I made was for example code in the documentation. Oops. Well, if you wanna fix it, that's cool. If not, I'll just go ahead and merge, haha.

**Comment by DovidM** [comment]
Got it

**Comment by kylefarris** [comment]
Thanks!

**Comment by kylefarris** [comment]
Hey Dovid,

I got caught up with other things at work and never published to NPM until now. The new version is 1.0.0 since the changes you made could represent breaking changes as it eliminates the chances of this module working on some older/supported versions of NPM.

Thanks again for your efforts!

Kyle

**Pull request merged**